docs/conversion_examples_gallery/recording/spikeglx.rst

```diff
@@ -13,7 +13,7 @@ Convert spikeglx data to NWB using :py:class:`~neuroconv.datainterfaces.ecephys.
     >>> # For this interface we need to pass the location of the ``.bin`` file
     >>> file_path = f"{ECEPHY_DATA_PATH}/spikeglx/Noise4Sam_g0/Noise4Sam_g0_imec0/Noise4Sam_g0_t0.imec0.ap.bin"
     >>> # Change the file_path to the location in your system
-    >>> interface = SpikeGLXRecordingInterface(file_path=file_path, verbose=False)
+    >>> interface = SpikeGLXRecordingInterface(file_path=file_path)
```

Comment:
Always thought it was a bit awkward as well to ask a first-time user to disable verbosity, when verbosity exists to allow human-readable communication with the user (not that it does much of this besides confirmation of validations at the NWBConverter level and success return at end of all `run_conversion` commands, but principle is it could always be enhanced).

Reply:
honestly, I'd prefer verbose to be False by default and to not set it to True here. I think verbose should really only be used for debugging purposes if the code does not work as expected in non-verbose mode. This is similar to how sklearn uses it. See e.g. https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html